### DISCUSSION TITLE
NBomber stalls when invoked by dotnet test runner in linux container

### DISCUSSION THREAD

**Discussion by madameczek** [VIP-Logic/NBomber#429] [Q&A]
Hi, 
I wrapped NBomber with xUnit tests. 
If I run tests from windows machine it goes well and assertions are performed. 
But when invoked in GitLab pipeline as test project (dotnet test), then NBomber stalls after welcome graphics and 2 more logs. The last log is `xxxxxx [INF] NBomber started as single node.` and nothing more happens. Test runner kills thread if I add timeout option `--blame-hang-timeout <TIMESPAN>` to `dotner test` invocation.
Image used by Gitlab executor is `image: mcr.microsoft.com/dotnet/sdk:5.0-focal`.

BTW. Do you consider to add test XML report with metrics, which could be used by Gitlab pipeline?
Marek

**Comment by AntyaDev** [comment]
Thanks for info.
Your PR with fixes and XML support is welcome!
